doc/user_manual/postprocessor.tex

```diff
@@ -149,6 +149,19 @@ \subsubsection{BasicStatistics}
   quantities are going to be calculated, if \textit{False} unbiased.
   \default{False}
   %
+  \item \xmlNode{dataset}, \xmlDesc{string (boolean), optional field}, if \textit{True} \xmlString{DataSet}
+    will be used to store the calculation results, if \textit{False} \xmlString{PointSet} or \xmlString{HistorySet}
+    will be used to store the calculation results.
+    \nb The optional \xmlString{DataSet} is added only to this Post-Processor, one can still use the \xmlString{OutStreams}
```

Comment:
```Post-Processor``` -> ```PostProcessor``` (for consistency regarding the naming convention)

Reply:
changed